demo-app/src/app/native-css-class/native-css-class.component.html

```diff
@@ -0,0 +1,68 @@
+<h3>Native CSS Class</h3>
+
+<div class="section">
+  <p class="col s12 m10">
+    Pages from Materialize that explain how to use their native CSS Class which can be used all around your application. <strong>Those CSS class will not be wrapped into directive.</strong>
```

Comment:
This paragraph is confusing .. I would suggest: Materialize CSS classes can be used all around your application. For this reason <strong>those CSS classes will not be wrapped into directive.</strong>